.github/workflows/rspack-update-tests-manifest.yml

```diff
@@ -41,7 +41,11 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN_PULL_REQUESTS }}
           BRANCH_NAME: rspack-manifest
-          SCRIPT: test/build-rspack-dev-tests-manifest.js
+          # We need to use `--override` for rspack (but not for turbopack).
+          # We don't currently have any CI running on every PR, so it's quite
+          # possible for us to regress on tests. We need to skip the
+          # only-promote-to-passing merge logic.
+          SCRIPT: test/update_build_manifest.js --bundler rspack --testSuite dev --override
```

Comment:
This was causing a small number of tests on arewerspackyet to show as passing when they are no longer passing.